CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ All notable changes to this project will be documented in this file. This projec
 * `isDragging` is now true whether or not something was picked in `onPointerMove()` event
 * Edit handles will now only appear in `modify` and `drawPolygon` modes
 * Can add new intermediate points anywhere along a line rather than just from the midpoint
+* `EditableGeoJsonLayer`'s `drawAtFront` prop should now be supplied via `modeConfig` prop
```

Comment:
Actually, this should go under the "Unreleased" section. Just uncomment that section header.